pages/portfolio.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Sophia's Personal Portfolio Site</title>
+  <link href="../stylesheets/style.css" rel="stylesheet">
 </head>
+
 <body>
-  
+
+  <header>
+    <p class="sitetitle">SOPHIA WAGNER</p>
+    <nav class="topnavigation">
+      <ul>
+        <li><a href=index.html>HOME</a> </li>
+        <li><a href=about.html>ABOUT</a> </li>
+        <li><a href=portfolio.html>PORTFOLIO</a></li>
+      </ul>
+    </nav>
+  </header>
+
+
+    <h1>PORTFOLIO</h1>
+    <main class = "innerflex">
+      
+      <figure class = "fleximage">
+        <img src="../images/television-cabinet.png" alt="Viewing Party Project" class="portfolio-image">
+        <figcaption>This <a href="https://github.com/sophwag/viewing-party">project</a> builds a system for tracking movie recommendations.</figcaption>
+      </figure>
+      <figure class = "fleximage">
+        <img src="../images/checklist.png" alt="Task List Project" class="portfolio-image">
+        <figcaption>This <a href="https://github.com/sophwag/task-list-api">program</a> tracks tasks and goals, and integrates with Slack.</figcaption>
+      </figure>
+      <figure class = "fleximage">
+        <img src="../images/toy-alphabet-blocks.png" alt="Adagrams Project" class="portfolio-image">
```

Comment:
Nice images! There are sites such as [Font Awesome](https://fontawesome.com/icons) where you can add icons directly as HTML and not as images. Just an extra suggestion to explore if you want!